gxa/src/test/java/uk/ac/ebi/atlas/experimentimport/analyticsindex/SolrInputDocumentInputStreamIT.java

```diff
@@ -91,14 +114,15 @@ public void testSomeExperiments() throws Exception {
         assertThat(count, is(greaterThan(100)));
```

Comment:
I realise this is an older bit of code and out of scope for this PR but this assertion is a bit odd... Why 100? Also, I don't get the point of the `count` variable. Why not just do `result.size()` instead?

Reply:
I’ll change it to 0.

As for your second question, `SolrInputDocument` only has iterator-based methods. There’s no way to get the size of the result other than by counting.